ADAL/src/ADAuthenticationContext.m

```diff
@@ -227,7 +227,8 @@ + (ADAuthenticationContext*)authenticationContextWithAuthority:(NSString*)author
 + (BOOL)isResponseFromBroker:(NSString *)sourceApplication
                     response:(NSURL *)response
 {
-    return response && [sourceApplication isEqualToString:ADAL_BROKER_APP_BUNDLE_ID];
+    return response && ([sourceApplication isEqualToString:ADAL_BROKER_APP_BUNDLE_ID]
```

Comment:
Can we provide a build flag for people to turn this on? I'd prefer for this dogfood behavior to only be for Microsoft dogfood apps.

Reply:
Good idea, done